partiql-planner/src/testFixtures/resources/tests/plan-typer/plan-typer-tests-ported.ion

```diff
@@ -0,0 +1,6717 @@
+
+test::{
+  name:"Pets should not be accessible #1",
+  type:"type",
+  body:{
+    statement:"SELECT * FROM pets",
+    session:{
+      catalog:"pql",
+      cwd:[
+      ]
```

Comment:
The vast majority of the typing tests use an empty session directory list. Wonder if we could simplify the logic by making a non-specified `cwd` equivalent to an empty session directory list.

Reply:
I like this! I'll update.